### DISCUSSION TITLE
Flutter Stripe <-> Stripe

### DISCUSSION THREAD

**Discussion by brnunes-stripe** [flutter-stripe/flutter_stripe#346] [General]
Hello, I work on the Android SDK for Stripe, we'd like to have a channel of communication with the maintainers of this project to coordinate releases.
We used to have Slack but now that links to a [Discord channel](https://discord.com/channels/841573134531821608/847727526985072681) which we don't have access.
Could you send us an invite to that Discord to [edit:email]?

**Comment by jonasbark** [comment]
Hi Bruno - I'll make sure you're getting invited to the Discord channel. It should be the best place for this.
